contract/security/ir.model.access.csv

```diff
@@ -9,3 +9,5 @@
 "contract_line_portal","Recurring portal","model_contract_line","base.group_portal",1,0,0,0
 "contract_template_line_manager","Recurring manager","model_contract_template_line","account.group_account_manager",1,1,1,1
 "contract_template_line_user","Recurring user","model_contract_template_line","account.group_account_invoice",1,0,0,0
+"contract_modification_user","Recurring user","model_contract_modification","account.group_account_invoice",1,1,1,1
```

Comment:
The name is not very descriptive (although it has been used constantly before). It's better to put:
```suggestion
"contract_modification_user","Contract modifications - User","model_contract_modification","account.group_account_invoice",1,1,1,1
```